website/content/docs/commands/hcl2_upgrade.mdx

```diff
@@ -98,9 +98,11 @@ working on improving this part of the transformer.
 
 ## Options
 
-- `-output-file` - File where to put the hcl2 generated config. Defaults to
-  JSON_TEMPLATE.pkr.hcl
-- `-with-annotations` - Adds helper annotations with information about the generated HCL2 blocks.
+- `-output-file` - Filename of the hcl2 generated config. Defaults to
```

Comment:
`Filename of the Packer `hcl2_upgrade` generated HCL2 template...` seems less confusing to me.  These are either config files or template files, and since Packer does have a config file it seems reasonable to use template file to describe what is being converted.

Reply:
```suggestion
- `-output-file` - Filename of the hcl2 generated template. Defaults to
```